source/comply/audit-log.rst

```diff
@@ -24,11 +24,17 @@ Configure audit logging
 
 Configuring Mattermost to enable audit logging requires editing the ``config.json`` file directly. Audit logging can’t be managed using the System Console.
 
-In the ``config.json`` file, go to the ``ExperimentalAuditSettings`` section. Within the ``AdvancedLoggingConfig`` setting, you can specify an absolute or relative filespec to another configuration file or a JSON string. The process of configuring audit logging includes specifying destination targets, event names to include, and the verbosity of the audit log output.
+In the ``config.json`` file, go to the ``ExperimentalAuditSettings`` section. Within the ``AdvancedLoggingConfig`` setting, you can specify an absolute or relative filespec to another configuration file, embed multi-line JSON in the ``config.json`` file, or an environment variable. The process of configuring audit logging includes specifying destination targets, event names to include, and the verbosity of the audit log output.
```

Comment:
`AdvancedLoggingConfig` is now deprecated and will be removed in a future release.  The new field is `AdvancedLoggingJSON` and only that field allows multi-line JSON.

AdvancedLoggingConfig (deprecated):  escaped single line JSON string  OR  filespec
AdvancedLoggingJSON:  multi-line JSON  OR  filespec   (allow supports the escaped single line JSON string but I don't think that should be documented)

Reply:
I completely missed that important nuance. Thank you, @wiggin77!